src/main/resources/application.yaml

```diff
@@ -50,3 +54,5 @@ spring:
   datasource:
     url: jdbc:postgresql://localhost:${DRAFT_STORE_DB_PORT:5432}/draftstore
     password: ${DRAFT_STORE_DB_PASSWORD:}
+idam:
+  useStub: true
```

Comment:
shouldn't you use the real one in functionalTests to make sure it works properly?

Reply:
This profile is used in only one weird test (`HandleUnknownExceptionTest`) where repo is actually mocked and set up to throw an exception. Not really a functional test.

I'll look into it on a separate occasion.